GatewayMicroService/GatewayMicroService/config/settings.py

```diff
@@ -0,0 +1,9 @@
+from pydantic_settings import BaseSettings
+import os
+
+
+class Appconfig(BaseSettings):
+    app_name: str = os.getenv("APP_NAME")
+    debug: bool = False
+    database_url: str = "postgresql://"+os.getenv("DB_USER")+":"+os.getenv("DB_PASSWORD")+"@"+os.getenv("DB_HOST")+"/"+os.getenv("DB_NAME")
```

Comment:
use an f string and make it configurable and get a getter method for it

Reply:
it need a getter method for each field of the App config?